### PR TITLE
chore: add missing struct so tests don't break

### PR DESCRIPTION
**Pull request by bvtujo**
I thought I fixed things before but I missed a struct that I hadn't added in the template. 

Turns out that golang templates don't like non-existent fields. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by huanjani — APPROVED** [review]

File: internal/pkg/template/workload.go [hunk 1]
```diff
@@ -117,6 +118,12 @@ type Volume struct {
 	IAM           *string // ENABLED or DISABLED
 }
 
+// AccessPointInfo holds information about how to create Copilot-managed access points
+type AccessPointInfo struct {
+	Uid *string
+	Gid *string
```
Comment (efekarakus) [inline internal/pkg/template/workload.go]: Can we rename the fields here and in the template to match https://github.com/golang/go/wiki/CodeReviewComments#initialisms?
```suggestion
	UID *string
	GID *string
```

Comment (bvtujo) [reply internal/pkg/template/workload.go]: I've adopted a more thoughtful struct name for this, which will be easier to extract from the manifest and inject into the template, and addressed the capitalization.

File: internal/pkg/template/workload.go [hunk 2]
```diff
@@ -117,6 +118,13 @@ type Volume struct {
 	IAM           *string // ENABLED or DISABLED
 }
 
+// CopilotVolumeCreationInfo holds information about how to create Copilot-managed access points
```
Comment (efekarakus) [inline internal/pkg/template/workload.go]: nit: period at the end of the sentence 🤦‍♂️

File: internal/pkg/template/workload.go [hunk 3]
```diff
@@ -117,6 +118,13 @@ type Volume struct {
...
+type CopilotVolumeCreationInfo struct {
```
Comment (efekarakus) [inline internal/pkg/template/workload.go]: What do you think of 
```suggestion
type ManagedVolumeCreationInfo struct {
```

**Review by efekarakus — APPROVED** [review]

**Pull request merged**